src/main/java/edu/cornell/kfs/fp/document/service/impl/CuDisbursementVoucherTaxServiceImpl.java

```diff
@@ -80,4 +86,48 @@ protected boolean validateNRATaxInformation(DisbursementVoucherDocument document
         }
         return true;
     }
+    
+	public boolean isForeignVendorWithTaxReviewRequired(String payeeTypeCode,
+			String paymentReasonCode, Integer vendorHeaderId) {
+
+		return isForeignVendor(payeeTypeCode, vendorHeaderId) && !paymentReasonDoesNotRequireTaxReviewForForeignVendor(paymentReasonCode);
+	}
+	
+	public boolean isForeignVendoAndNOTaxReviewRequired(String payeeTypeCode,
+			String paymentReasonCode, Integer vendorHeaderId) {
+
+		return isForeignVendor(payeeTypeCode, vendorHeaderId) && paymentReasonDoesNotRequireTaxReviewForForeignVendor(paymentReasonCode);
+	}
+	
+	protected boolean isForeignVendor(String payeeTypeCode, Integer vendorHeaderId){
+		if (payeeTypeCode.equals(KFSConstants.PaymentPayeeTypes.VENDOR) && vendorService.isVendorForeign(vendorHeaderId)){ 
+			return true;
+		}
+		
+		return false;
```

Comment:
In this method and the one below it, I would suggest returning the result of the condition so that you don't need the extra "return true/false" statements.